raven/README.md

```diff
@@ -91,6 +94,11 @@ The _putPreference_ rules are optimizations; they do not restrict the destinatio
 where a namespace is intended to be stored only in some site(s) or where most or all PUTs come from systems that are near one 
 storage site. TODO: support for `GET` preferences? also use client IP address to prioritize?
 
+The _putAvoid_ key configures raven to **never** return PUT URLs to the specified site(s).
+
+The _getAvoid_ key configures raven to **only** return GET URLs to the specified site(s) if the site contains the only copy
```

Comment:
"only copy"? Can it return other sites in this case?

Reply:
Yes, the idea is that raven will only output a URL to such sites if the artifact is only available there (that's not exactly _the only copy_ since _available_ also includes transient issues like another site being down... but it minimises the number of GETs to a site

If other sites have an (available) copy, then a `getAvoid` site will not be used

Reply:
Thanks. To me the last explanation makes more sense - avoid to GET from this sites unless no other copies exist.